docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,138 @@
+letters within the parentheses represent the state of a parachain block. Hence **C** (candidate,
+created by collators), **B** (backed, by para-validators), and **I** (included, by block authors in
+the relay-chain). For more information about parablock validation see the
+[parachain protocol page](./learn-parachains-protocol.md).
+
+In the diagram there are three relay-chain blocks being finalized (**R1-2**) and a parachain
+(**Parachain 1**) proposing two parablocks (**P1** and **P2**). Each parablock's lifecycle, from
+being candidate (**P(C)**) to being backed and included (**P(B)** and **P(I)**, respectively), must
+fit within the relay-chain block lifetime.
+
+Thus, the main limitation of synchronous backing is that parablock validation is tightly coupled to
+the relay chain's progression on a 1-1 basis, meaning every parablock must be built, backed, and
+included within six seconds. Parablocks depend on being in sync with the relay chain, reducing the
+amount of data in the block.
+
+Essentially, a parablock is rushing to being validated by the relay-chain due to this synchrony.
+
+By making this process of backing parablocks more asynchronous, parachains get the chance to not
+only include more data within each parablock, but also retry to include parablocks that failed
+inclusion.
+
+## Asynchronous Backing on Polkadot
+
+With asynchronous backing, the window is more than the span of around two blocks, or a ~12-second
+window. This enables more computational and storage time per block, as the context of the next relay
+chain block can kickstart the process of the next parablock.
+
+Notice that blocks can be prepared longer, meaning more transactions per block. Due to the
+asynchrony, these blocks can be prepared in anticipation of being included later rather than keeping
+in sync with the relay chain's progress 1-1:
+
+```mermaid
+%%{init: { 'logLevel': 'debug', 'theme': 'neutral', 'themeVariables': { 'fontSize': '14px', 'commitLabelFontSize': '16px', 'tagLabelFontSize': '16px' }, 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchName': 'Relay Chain'}} }%%
+gitGraph
+commit id:"R1"
+branch "Parachain 1, Block 1" order: 2
+commit id:"P1(C)"
+commit id:"P1(B)"
+branch "Parachain 1, Block 2" order: 3
+checkout "Relay Chain"
+merge "Parachain 1, Block 1" tag:"P1 Backed" type:HIGHLIGHT
+commit id:"R2"
+checkout "Parachain 1, Block 2"
+commit id:"P2(C)"
+commit id:"P2(B)"
+checkout "Relay Chain"
+merge "Parachain 1, Block 2" tag:"P2 Backed" type:HIGHLIGHT
+checkout "Parachain 1, Block 1"
+commit id:"P1(I)"
+checkout "Relay Chain"
+merge "Parachain 1, Block 1" tag:"P1 Included" id:"R3"
+commit id:"R4"
+checkout "Parachain 1, Block 2"
+commit id:"P2(I)"
+checkout "Relay Chain"
+merge "Parachain 1, Block 2" tag:"P2 Included" id:"R5"
+```
+
+:::info How does this compare to Ethereum's Danksharding?
+
+Asynchronous backing is the Polkadot equivalent to Ethereum's Danksharding. Where Ethereum merely
```

Comment:
Was this backed by Rob or the authors of Async backing anywhere or is it our interpretation?

```suggestion
Asynchronous backing is the Polkadot (sort of) equivalent to Ethereum's Danksharding. Where Ethereum merely
```

Reply:
It was my interpretation, but I think it should probably be taken out by now.  Maybe later we can include it in a comparison where it really belongs

Reply:
Sounds good to me! Thanks